### PR TITLE
Storybook v6

### PR DESCRIPTION
**Pull request by nicolls1**

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx
Comment (nicolls1) [inline src/components/Checkbox/Checkbox.tsx:318]: Instead of using mdx we can just add comments in the code and it will pick them out and add it to the storybook. In general, I think is self explanatory and we don't need as much text.

File: package.json
Comment (MMFane) [inline package.json:45]: If we're not using knobs anymore, should this import be removed?

**Review by MMFane — APPROVED** [review]
Looks good to me! I'm excited to have to write less for each story

File: package.json
Comment (nicolls1) [reply package.json:45]: Good call, I'm still running the storybook with the old components included, will remove at the end

**Comment by nicolls1** [comment]
The correct semantic commit for this would be `docs` correct?

**Review by xdumaine — APPROVED** [review]
This will need another approver after me since I merged a small fix for the build.

**Review by HaleyWardo — APPROVED** [review]
Wow! The docs look way better 🤩 ! Thank you for making this huge updat!

**Pull request merged**
